src/content/support/p1-process/index.md

```diff
@@ -0,0 +1,90 @@
+
+- Alert received pointing to a impaired production system
+- Customer sent us a urgent email 
+- Customer reached us via Slack
+
+Once our engineer acknowledge the message we need to confirm the severity to trigger the P1 process. In case the customer paged us, the Giant Swarm engineer joins the call or thread to verify the severity. In case an alert paged us, the engineer connects to the platform and measure the impact.
+
+Once the criticality is confirmed we continue the workflow by [declaring an Incident](https://help.incident.io/en/articles/5947915-declaring-incidents), which creates a Slack incident channel and triggers a custom [workflow](https://help.incident.io/en/articles/6971329-getting-started-with-workflows) that drives the engineer through the next steps. 
+
+When the incident is created there are some data the engineer must fill in (short summary, severity and customer impacted). When severity is `Critical` (P1 incident) the first thing to do is building the team. Most of the times the engineer creating the channel is not part of the [Incident Coordinators Group]() so he needs to escalate it to get someone from that team. For that reason, when you create an incident channel, incident.io show you a button to escalate. In the end, we need at least two person team to manage a critical incident (communications and operations).
+
+### Investigate
+
+Once team is built the person assigned to the Operations Engineer role will carry on with the investigation. The Incident Coordinator will be in contact with the customer, via messaging or in a call, and will provide information to the Operator to help with the investigation.
+
+We leave space to the Operations Engineers to focus on the investigation but we establish 20 minutes periods to get back and inform on the state. Most of the times the OEs share the findings in the channel and IC pin those message to help tracking the actions performed.
+
+In case the Incident Coordinator realize we need to grow the group, he can always escalate to more memebers of the team using incident.io commands.
+
+### Fixing
+
+After we have found the root cause of the problem we implement a solution, which can be temporal, to avoid any more damage the customer service.
+
+### Monitoring
+
+Once the fix or workaround has been implemented, we communicate with the customer and move to a monitoring phase, where we stay in stand-by keeping an eye on the metrics and communication channel to confirm there is no a regression coming. We leave the incident in this state for some time, could be a day or two, till we agree with the customer there is no regression.  
+
+### Closing up
+
+When we close the incident the work is not yet finished. We write down a [PostMortem]() document to detail all information collected through the incndent and share with the customer. We usually create follow ups that convert in tickets to our product teams so we can improve our service and avoid same mistake twice.
```

Comment:
Who is responsible of writing the postmortem?

Reply:
Maybe some collaborative effort?
- Incident coordinator writes the high overview.
- One or more operations engineers write deeper details about the issues they have debugged and fixed.